### DISCUSSION TITLE
Offline status in Explorer's Stake & Delegate tab

### DISCUSSION THREAD

**Discussion by rufatnas23** [subquery/subql#1002] [Frontier TestNet - Bug Reporting]
Indexing is running.  In the List of Indexers the Status is Ready, but in Explorer Stake&Delegate Bar status is offline, is it ok or a bug ?
![ae4484c1-ef8c-4b9e-9aab-971a431fa776](https://user-images.githubusercontent.com/94823027/167563273-571575eb-0fc2-422c-96e9-ff92dd8cb9fb.jpg)

**Comment by mzxyz** [comment]
For upgrade indexer services, normally not necessary to reindexing the project unless we publish versions with breaking change. 
For your question, the problem is that you were indexing these project before and reported the project status to network with a higher block height, the coordinator will only report the project status until you current indexing block height catch up with the one on the Network (contract). Before the status update, the network found your service stop reporting the block height for a while than the status will be offline
